documentation/PPIData.md

```diff
@@ -0,0 +1,27 @@
+[Back to main page](../README.md)
+
+# PPI Data in Polar devices
+
+Pulse-to-Pulse interval is the signal between two main peaks of the Photoplethysmography (PPG) signal obtained by optical heart rate sensor used in Polar wrist devices or optical HR sensors such as Verity Sense and OH1.
+
+It is very similar to the R-R interval between two R peaks of QRS complex calcualted by electrocardiography (ECG) based sensors (H9/H10) only not in the same phase.
+
+The other main difference is that R-R intervals that is much easier to detect with ECG based sensors, as the R wave peaks have usually a great amplitude compared to the rest of the signal as long as the strap is moisturized and has good contact with the skin. 
+
+On the other hand, 
+
+> [!IMPORTANT]
+> 
+> P-P intervals are much more sensitive to noise (movement, light leaking if not tight enough, etc.) compared to R-R intervals. It means that this metric **cannot** be measured accurately during activity. It should only be used at complete rest.
+
+Each PPI sample that can be fetched using Polar BLE SDK has 2 important flags : 
+
+- The skin contact flag : indicates if there was skin contact present for that sample
```

Comment:
imho worth noting that Verity Sense and similar sensors do not support skin contact flag

Reply:
@orestesgaolin Well now that you say it, I see that the PPI packets from Verity Sense have skin_contact_support = 1 and the skin contact bit to 1 even device is on the table, which is quite in my opinion. Skin contact support should be 0 as its so unreliable.  I'll make a note about this in Verity Sense specific document.